src/BootstrapForm.php

```diff
@@ -143,13 +143,19 @@ protected function model($options)
         // If the form is passed a model, we'll use the update route to update
         // the model using the PUT method.
         if (!is_null($options['model']) && $options['model']->exists) {
-            $route = Str::contains($options['update'], '@') ? 'action' : 'route';
-
-            $options[$route] = [$options['update'], $options['model']->getRouteKey()];
+            $update = is_array($options['update']) ? $options['update'][0] : $options['update'];
+            $route = Str::contains($update, '@') ? 'action' : 'route';
+            
+            if (is_array($options['update'])) {
```

Comment:
Would it work if you just cast `$options['update']` to an array and then merge it? Would save you needing the if/else statement.

`$options[$route] = array_merge((array) $options['update'], [$options['model']->getRouteKey()]);`